### DISCUSSION TITLE
On Release If Conditional

### DISCUSSION THREAD

**Discussion by cheatmenot** [community/community#13137] [Actions]
I tried to add if conditional on release but it doesn't work, I think there are lot of possibilities with adding filter on release

on:
  release:
    if: {{ expression }}
    types: [published]

**Comment by ethomson** [comment]
Thanks @cheatmenot - at present we do not support conditionals at that level, but I agree that there's a lot of opportunities here to improve this. We'll put this on the backlog. Thanks for the feedback!
